docs/robots/solutions/husky_observer/user_manual_husky_observer.mdx

```diff
@@ -1386,20 +1386,118 @@ to contact Clearpath if you have any questions.
 
 :::
 
-### FLIR PTZ Camera Configuration
+### Axis PTZ Camera Configuration
+
+The Husky Observer is equipped with an [Axis Q62](https://www.axis.com/products/axis-q62-series) series PTZ camera, with
+colour and low-light capabilities, defogger, and lens wiper.
 
 #### Key Settings
 
-- Resolution: TODO
-- Frame rate: TODO frames per second (front); TODO frames per second (rear)
+- Resolution: Max: 1920x1080.  ROS driver defaults: 640x480
+- Frame rate: 60Hz
 
 #### Where to make adjustments
 
-- TODO
+Global camera settings (like pan & tilt limits) can be set by logging into the Axis setup tool on the camera. Connect
+a laptop to the robot and assign the laptop a static IP address in the range `192.168.131.100-200`.  Then open a
+browser and navigate to [http://192.168.131.10](http://192.168.131.10).
+
+ROS driver settings can be edited in `/etc/ros/noetic/ros.d/axis_ptz.launch` or by changing the environment variables
+found in `/etc/ros/setup.bash` on the robot's primary computer.
 
 #### Pan Limits
 
-TODO: Chris to add his notes here
+The Axis Q62 is normally capable of making continuous rotations in either direction. This allows the camera to always
+take the shortest trajectory to reach any given pan angle. However, if you have additional equipment mounted on top of
+the camera, for example a Flir thermal camera or a microphone, this behaviour may cause the wires to get tangled around
+the PTZ camera and potentially cause damage to the robot.
+
+To prevent damage to the robot, Clearpath recommends setting pan limits on the Axis camera.  This will restrict the
+camera's range of motion to less than 180 degrees in either direction, eliminating the risk of the camera making
+continuous rotations.
+
+To set pan limits, connect your laptop to one of the robot's exposed ethernet ports and assign your laptop a static
+IP address in the range `192.168.131.100-200`.  Open your web browser and navigate to
+[http://192.168.131.10](http://192.168.131.10). Log into the Axis configuration website using the username `root` and
```

Comment:
Can we refer them to Appendix A for the credentials? Trying to keep all that info in one place if possible.

Reply:
Done.